user/getting-started.md

```diff
@@ -14,197 +13,130 @@ information from your code.
-Notice how in this case we need to share the host Docker server's Unix socket with
-the container (`-v /var/run/docker.sock:/var/run/docker.sock`) so it can access it
-to retrieve the local images.
+The binary *bblfshd* is provided with a sister tool called *bblfshctl*, this
+command is used to monitor and manage the daemon.
 
-Or if you prefer running it in standalone mode:
+If you are using the docker image this binary is inside of the `bblfsh/bblfshd`
+image and can be using with a `docker exec`.
 
-```
-$ BBLFSH_DRIVER_IMAGES="python=docker-daemon:bblfsh/python:dev-96b24d3;java=docker-daemon:bblfsh/java-driver:latest" bblfsh server
-DEBU[0000] binding to 0.0.0.0:9432                      
-DEBU[0000] initializing runtime at /tmp/bblfsh-runtime  
-DEBU[0000] Overriding image for python: docker-daemon:bblfsh/python:dev-96b24d3
-DEBU[0000] Overriding image for java: docker-daemon:bblfsh/java-driver:latest
-DEBU[0000] starting server                              
-DEBU[0000] registering gRPC service                     
-INFO[0000] starting gRPC server
+```sh
+$ docker exec -it bblfshd bblfshctl --help
 ```
 
+```sh
+Usage:
+  bblfshctl [OPTIONS] <command>
 
-### Setting maximum message size
+Help Options:
+  -h, --help  Show this help message
 
-If a customized [gRPC](https://grpc.io) message size is needed, you can use the command flag option `--max-message-size`.
-By default 100MB is current upper limit, but you can override it:
-
-    docker run --privileged -p 9432:9432 --name bblfsh bblfsh/server --max-message-size=500
-
-or running the Babelfish Server in local:
-
-    sudo bblfsh server --max-message-size=500
-
-The number given to the `--max-message-size` option represents the size in MB, and it defines the limit for both directions: send and receive.
-
-
-## Babelfish Tools
-
-Babelfish Tols provide some language analysis tools on top of Babelfish. You can
-use them for various purposes:
-
-- Check that the server is working properly
-- Get some data from the source code
-- See how a language analysis tool is implemented, as a basis for your own
-  tools.
-
-
-### Setup
-
-Running Babelfish Tools standalone requires getting the `bblfsh-tools`
-binary. Currently this requires a working setup of
-[Go](https://golang.org/doc/install). You can get it with the following command:
-
-```bash
-$ go get -u github.com/bblfsh/tools/...
+Available commands:
+  driver     Manage drivers: install, remove and list
+  instances  List the driver instances running on the daemon
+  parse      Parse a file and prints the UAST or AST
+  status     List all the pools of driver instances running
 ```
 
-### Usage
-
-Babelfish Tools provides a set of tools built on top of Babelfish, to
-see which tools are supported, run:
-
-`bblfsh-tools --help`
+### Driver management
 
-There's an special tool, the `dummy` tool, which should let you know if the
-connection with the server succeeded:
+The *bblfshd* driver can be install, updated and remove with the `driver`
```

Comment:
It should be "bblfshd's drivers can be installed..." or "A bblfshd driver can be installed..."